kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -22,6 +22,8 @@ spec:
         ports:
         - name: http-metrics
           containerPort: 8080
+        - name: telemetry-metrics
```

Comment:
From the build failure:

> The Deployment "kube-state-metrics" is invalid: spec.template.spec.containers[0].ports[1].name: Invalid value: "telemetry-metrics": must be no more than 15 characters

I'd suggest `self-metrics`.

Reply:
I have changed the port name to `telemetry`. :)